source/deprecations/ember-data/v2.x.html.md

```diff
@@ -391,20 +388,61 @@ let post = this.store.peekRecord('post', 123);
 post.serialize();
 ```
 
+### Deprecations Added in Ember Data 2.12
+
 #### JSONSerializer.shouldSerializeHasMany
 
 ###### until: 3.0.0
 ###### id: ds.serializer.private-should-serialize-has-many
-###### feature: ds-check-should-serialize-relationships
 
 The private method `_shouldSerializeHasMany` has been promoted to the public
 API. To remove this deprecation, please remove the underscore to use the public
 [`shouldSerializeHasMany`](http://emberjs.com/api/data/classes/DS.JSONSerializer.html#method_shouldSerializeHasMany)
 method.
 
-#### HasManyReference.push(array)
+### Deprecations Added in Ember Data 2.15
```

Comment:
Should this say `Deprecations Added in Ember Data 2.13`?

Reply:
Yes thanks, I had a local change for it 👍